src/Microsoft.OData.Core/ODataMessageWriterSettings.cs

```diff
@@ -215,6 +215,12 @@ internal string AcceptableMediaTypes
             }
         }
 
+        /// <summary>
+        /// The is IEEE 754 Compatible is used to determine if the message follows IEEE 754 standard.
+        /// If this is set to true then long and decimals will be serialized as strings.
+        /// </summary>
+        internal string IsIeee754Compatible { get; set; }

```

Comment:
>string [](start = 17, length = 6)

bool?

Reply:
or bool? ?

---
In reply to: [375396724](https://github.com/OData/odata.net/pull/1659#discussion_r375396724) [](ancestors = 375396724)

Reply:
Here i am think it's better to use "bool?" or Nullable<bool> . So we can distinguish between
1) set from customer
2) not set from customer

---
In reply to: [375396872](https://github.com/OData/odata.net/pull/1659#discussion_r375396872) [](ancestors = 375396872,375396724)